arangod/RestHandler/RestRepairHandler.cpp

```diff
@@ -314,10 +314,10 @@ ResultT<bool> RestRepairHandler::jobFinished(std::string const& jobId) {
         << "Failed to get job status: "
         << "[" << jobStatus.errorNumber() << "] " << jobStatus.errorMessage();
 
-    return std::move(std::move(jobStatus).result());
+    return ResultT<bool>::error(std::move(jobStatus.result()));
```

Comment:
```suggestion
    return ResultT<bool>::error(std::move(jobStatus).result());
```

Reply:
@mpoeter I did the change with the move on purpose, not by accident. If you see think this isn't right this way, please elaborate.

The move is actually also really unnecessary here, this place is as far from being performance critical as it gets.

Reply:
The move was applied on `jobStatus` so that the ref-qualified `result()` overload is called; this was introduced in #8157. But I agree that it is not performance critical.

Reply:
Ah, I see. I've overlooked that there is no non-`const` `Result& result() ...` accessor that would allow the move I intended.